x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -54,11 +54,19 @@ export interface ComplianceDashboardData {
 export type CspStatusCode =
   | 'indexed' // latest findings index exists and has results
   | 'indexing' // index timeout was not surpassed since installation, assumes data is being indexed
+  | 'unprivileged' // user lacks privileges for the latest findings index
   | 'index-timeout' // index timeout was surpassed since installation
   | 'not-deployed' // no healthy agents were deployed
   | 'not-installed'; // number of installed csp integrations is 0;
 
+export type IndexStatus = 'exists' | 'empty' | 'unprivileged';
```

Comment:
`empty` means both `exists without data` and `none-exists` ? if so maybe lets add another case

Reply:
hmm yea we can improve that
`exists` - index contains data
`empty` - no data in index or index does not exist

maybe `exists` -> `has_data`
and `empty` -> `no_data`